tools/delete_candidate.php

```diff
@@ -26,264 +41,192 @@
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
 
+// Possible script actions
```

Comment:
This section seems unnecessary now. The only valid action is delete_candidate. Also makes execution a little clunky e.g. "Example: php delete_candidate.php **delete_candidate** 965327 dcc0007".

I'd remove the command line parameter and switch-case statements.